experimental/traffic-portal/src/app/core/parameters/table/parameters-table.component.ts

```diff
@@ -0,0 +1,146 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { Component, type OnInit } from "@angular/core";
+import { FormControl } from "@angular/forms";
+import { MatDialog } from "@angular/material/dialog";
+import { ActivatedRoute } from "@angular/router";
+import { BehaviorSubject } from "rxjs";
+import {ResponseParameter} from "trafficops-types";
+
+import { ParameterService } from "src/app/api";
+import { CurrentUserService } from "src/app/shared/current-user/current-user.service";
+import { DecisionDialogComponent } from "src/app/shared/dialogs/decision-dialog/decision-dialog.component";
+import type { ContextMenuActionEvent, ContextMenuItem } from "src/app/shared/generic-table/generic-table.component";
+import { NavigationService } from "src/app/shared/navigation/navigation.service";
+
+/**
+ * ParametersTableComponent is the controller for the "Parameters" table.
+ */
+@Component({
+	selector: "tp-parameters",
+	styleUrls: ["./parameters-table.component.scss"],
+	templateUrl: "./parameters-table.component.html"
+})
+export class ParametersTableComponent implements OnInit {
+	/** List of parameters */
+	public parameters: Promise<Array<ResponseParameter>>;
+
+	/** Definitions of the table's columns according to the ag-grid API */
+	public columnDefs = [
+		{
+			field: "id",
+			headerName: "ID"
+		},
+		{
+			field: "configFile",
+			headerName: "Config File"
+		},
+		{
+			field: "name",
+			headerName: "Name"
+		},
+		{
+			field: "profiles",
+			headerName: "Profiles"
+		},
+		{
+			field: "secure",
+			headerName: "Secure"
+		},
+		{
+			field: "value",
+			headerName: "Value"
+		},
+		{
+			field: "lastUpdated",
+			headerName: "Last Updated"
+		}
+	];
```

Comment:
None of these columns are hidden by default, but I'd argue most are pretty irrelevant to the use-case of your average user. The TPv1 table doesn't include ID or Last Updated at all, so those can probably be hidden. I also suggest hiding "secure" by default, since that's obvious from the value anyway (and hardly used at any rate)